depthai_helpers/utils.py

```diff
@@ -26,11 +26,11 @@ def to_tensor_result(packet):
     data = {}
     for tensor in packet.getRaw().tensors:
         if tensor.dataType == dai.TensorInfo.DataType.INT:
-            data[tensor.name] = np.array(packet.getLayerInt32(tensor.name)).reshape(tensor.dims)
+            data[tensor.name] = np.array(packet.getLayerInt32(tensor.name)).reshape(tensor.dims[::-1])
         elif tensor.dataType == dai.TensorInfo.DataType.FP16:
-            data[tensor.name] = np.array(packet.getLayerFp16(tensor.name)).reshape(tensor.dims)
+            data[tensor.name] = np.array(packet.getLayerFp16(tensor.name)).reshape(tensor.dims[::-1])
         elif tensor.dataType == dai.TensorInfo.DataType.I8:
-            data[tensor.name] = np.array(packet.getLayerUInt8(tensor.name)).reshape(tensor.dims)
+            data[tensor.name] = np.array(packet.getLayerUInt8(tensor.name)).reshape(tensor.dims[::-1])
```

Comment:
CC @themarpe on this, `tensor.dims` is returning tensor dimensions but in reverse order. For example, for `openpose2`, the output dimensions are
```
[1, 57, 46, 54]
```
whereas the `tensor.dims` returns
```
[54, 46, 57, 1]
```

It had no impact in deeplabv3p_person since it's output looks like
```
[256, 256, 1, 1]  # incorrect, current
[1, 1, 256, 256]  # expected
```

so after squashing it still ended up being `[256, 256]`

Added reverse here (`[::-1]`) but I think we should fix that in the API too

Reply:
Resolved in https://github.com/luxonis/depthai-core/pull/160
Either do a version check and use reverse if < 2.6.0, or we can update later on?

Reply:
I'd say to update it later on when we'll be bumping 2.6.0